core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableConvention.java

```diff
@@ -34,6 +34,13 @@
    * "typical" calling convention. */
   public static final double COST_MULTIPLIER = 1.0d;
 
+  /**
+   * User can switch this config to allow conversion by
+   * {@link org.apache.calcite.plan.volcano.AbstractConverter}, which is relied
+   * upon when converting collations/distributions of input to fit the parent node.
+   */
+  public static boolean useAbstractConvertersForConversion = false;
```

Comment:
This should be used for test purpose only, right? If yes, can you mark it with `For private use only, may be removed without notice`?
The value should not be dynamically changed.

Reply:
Never user the word "user" in documentation. There's almost always a better word.

Reply:
Thanks ~ I refined the doc to indicate this config is relied on by some tests.
I'm hesitate to put a `"it's only for test"` here. Because though `EnumerableConvention` provide an interface of `useAbstractConvertersForConversion(fromTraits, toTraits)`, when user asks if it's possible to run a Sort-Merge-Join in Enumerable convention, it's hard to answer. Because convention of all enumerable operators is `EnumerableConvention.INSTANCE`, though we give an interface there, it's hard to customize.